src/workloads/execution/MixedMultiDeletesBatched.yml

```diff
@@ -0,0 +1,37 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Deletes a range of documents using the BATCHED_DELETE query exec stage both in isolation and while performing writes on another collection. Tests deletes on documents of size ~1KB then deletes on documents of ~10MB.
+
+Keywords:
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+
+Clients:
+  Default:
+    QueryOptions:
+      # Allow for longer duration since the loading of the data may take a while.
+      socketTimeoutMS: 3_600_000  # = 1 hour
+
+LoadConfig:
+  Path: "../../phases/execution/MixedMultiDeletes.yml"
+  Parameters:
+    UseBatchedDeleteStage: 1
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - replica-all-feature-flags
+      - replica
+    branch_name:
+      $neq:
+      # 6.0 Does not have the 'batchUserMultiDeletes' server parameter.
+      - v6.0
```

Comment:
Hey, thanks for fixing this! One note. Not sure why we didn't get any BFs for other branches, but wouldn't this cause these workloads to still run for branches under 6.0? Such as 5.0, 4.4, 4.2?

Reply:
I believe it will only run our most recent major release, but I could be wrong.

Reply:
Not sure either, but the docs and the other workloads seem to be excluding every branch explicitly.

https://github.com/mongodb/genny/blob/c1536caf49eeddf49d31d0307e7f8a9cecf5034f/docs/using.md#L553

Reply:
Hm, okay, I'll modify it for down to 4.2 since we no longer support 4.0

Reply:
Thanks for being so attentive to detail!